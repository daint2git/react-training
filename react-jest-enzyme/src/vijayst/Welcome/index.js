import React from 'react'

export default class Welcome extends React.Component {
  render() {
    return <div>Hello world</div>
  }
}
