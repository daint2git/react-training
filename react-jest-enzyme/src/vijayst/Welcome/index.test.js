import renderer from 'react-test-renderer'
import Welcome from './'

describe('Welcome', () => {
  test('Snapshot', () => {
    const component = renderer.create(<Welcome />)
    const json = component.toJSON()
    expect(json).toMatchSnapshot()
  })

  test('Welcome renders "Hello world"', () => {
    const welcome = shallow(<Welcome />)
    expect(welcome.find('div').text()).toEqual('Hello world')
  })
})
