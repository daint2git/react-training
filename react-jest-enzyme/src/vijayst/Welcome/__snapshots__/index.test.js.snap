// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Snapshot 1`] = `
<div>
  Hello world
</div>
`;
